src/include/ndpi_protocol_ids.h

```diff
@@ -363,6 +363,7 @@ typedef enum {
   NDPI_PROTOCOL_TPLINK_SHP            = 332, /* TP-LINK Smart Home Protocol */
   NDPI_PROTOCOL_SOURCE_ENGINE         = 333,
   NDPI_PROTOCOL_BACNET                = 334,
+  NDPI_PROTOCOL_HOTS                  = 335, /* Heroes of the Storm */
```

Comment:
Please rebase to the current `dev` branch and change to:

```suggestion
  NDPI_PROTOCOL_HOTS                  = 336, /* Heroes of the Storm */
```
A recent merge occupied `335` already. :)

Reply:
Changed id.